admin/src/pages/CreationConfirm.vue

```diff
@@ -71,21 +72,6 @@ export default {
           this.toastError(error.message)
         })
     },
```

Comment:
whats with `confirmCreation`? Why can this mutation not live in the `apollo` field? Can we not annotate the corresponding button aswell?

Reply:
Docu says: Use `this.$apollo.mutate()` to send a GraphQL mutation.
See: https://apollo.vuejs.org/guide/apollo/mutations.html
I think we use it correctly